### DISCUSSION TITLE
Django 3.2 weird errors

### DISCUSSION THREAD

**Discussion by rohanahata** [encode/django-rest-framework#7923] [Potential Issue]
Have a url set up such as this:
```
@action(methods=['GET'], detail=False, url_name='scan_qr')
    def scan_qr(self, request, pk=None):
```
When I use 3.1.5 everything works fine. As soon as I switch to 3.2, this redirects me using a 302 and throws an error

**Comment by carltongibson** [comment]
@rohanahata — You need to provide **much** more detail. (For a start, what's the error? — including the traceback)
